.changeset/young-jobs-wink.md

```diff
@@ -0,0 +1,6 @@
+---
+"@medusajs/medusa": patch
+"integration-tests-api": patch
```

Comment:
comment: I don't think we should be including integration tests in our changesets, as they will not be versioned

Reply:
indeed, is it possible to ignore those "packages" from changeset?